core/examples/luigi-sample-angular/e2e/tests/product-switcher.spec.js

```diff
@@ -0,0 +1,18 @@
+describe('ProductSwitcher', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:4200');
+    cy.login('tets@email.com', 'tets');
+  });
+
+  it('Clicking around the product switcher', () => {
```

Comment:
Maybe here it would be good to click some of the links and check the location, similarly as we are doing on context-switcher.spec.js file.

Reply:
also fixed

Reply:
The test is not working anymore, at least in my local machine, could you please check? 
![image](https://user-images.githubusercontent.com/8238188/51675545-4274f600-1fd4-11e9-9441-baf83a29c806.png)

Reply:
The test is not working after the last changes, at least on my local machine. Could you please check? 
![screen shot 2019-01-24 at 12 33 15](https://user-images.githubusercontent.com/8238188/51675579-5caed400-1fd4-11e9-92c6-c12f2632e237.png)